src/data/sql-quiz.ts

```diff
@@ -1,12 +1,12 @@
 const sqlQuiz = [
   {
-    Question: "Which of the following is the most effective way to protect against SQL injection attacks?", 
-    Answer: "Using a modern SQL library that sanitizes SQL inputs",
-    Distractor1: "Using strong passwords for database access",
-    Distractor2: "Disabling certain database permissions",
-    Distractor3: "Encrypting database tables",
+    Question: "In terms of Big-O complexity, what is the time complexity for lookups in a binary tree index?", 
+    Answer: "O(log n)",
+    Distractor1: "O(n)",
+    Distractor2: "O(n log n)",
+    Distractor3: "O(1)",
```

Comment:
this question should be added to the general cs quiz.
But also, you shouldn't override the first question of the SQL quiz.
Please fix this. Thank you